Assets/MixedRealityToolkit-Examples/AdaptiveQuality/AdaptiveQualityExample.cs

```diff
@@ -2,22 +2,24 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using UnityEngine;
-using HoloToolkit.Unity;
 
-public class AdaptiveQualityExample : MonoBehaviour
+namespace MixedRealityToolkit.Examples.AdaptiveQuality
 {
-    public TextMesh Text;
-    public AdaptiveQuality Quality;
-
-    private void Update()
+    public class AdaptiveQualityExample : MonoBehaviour
     {
-        Text.text = string.Format("GPUTime:{0:N2}\nQualityLevel:{1}\nViewportScale:{2:N2}",
-            GpuTiming.GetTime("Frame") * 1000.0f,
-            Quality.QualityLevel,
+        public TextMesh Text;
+        public MixedRealityToolkit.Utilities.AdaptiveQuality.AdaptiveQuality Quality;
```

Comment:
Seems kinda long.

Reply:
Or, would it make more sense to add the `Examples` subnamespace at the end to make it all fall under the ` MixedRealityToolkit.Utilities.AdaptiveQuality` namespace?

Reply:
Agree, I didn't like these.  I agree, let's move these up and drop the unnecessary folders.

Reply:
Done.  Hopefully that will complete this phase of the namespace update.